### ISSUE TITLE
darknet: ./src/utils.c:256: error: Assertion `0' failed.

### ISSUE THREAD

**Issue by ivankg89** [pjreddie/darknet#1930]
Thanks to provide the source! Hope your reply!

I convert the exam yolov3-tiny successful, but when I convert my own yolov3 model, I recipe the message: First section must bu [net] or [network]: Success.
darknet2ncnn: ./src/utils.c: 256: error: Assertion '0' failed.
Aborted (core dumped).

**Comment by wutianze** [comment]
I have the same problem, I can run with my own weights and .cfg, but the initial weights doesn't work

**Comment by wutianze** [comment]
I solve this by replacing the yolov3.cfg with a new one, may this help you

**Comment by ivankg89** [comment]
Could you please send me your cfg?

**Comment by wutianze** [comment]
I use the yolov3.cfg in pjreddie/darknet

**Comment by neverstoplearn** [comment]
i meet the same error , have someone help me? thanks

**Comment by goldengrisha** [comment]
In my case, that was the problem with images width and height, please check.

**Comment by martinrkeyser** [comment]
Hey people. I had the same issue, in the end I had to update the ARCH line in my makefile and rebuild. Please note, I did have to install CUDA V10 because I am running a GTX 1650 using the compute_75 architecture.

**Comment by dm0mb** [comment]
> In my case, that was the problem with images width and height, please check.

I think you are right, I change width and height get this issue.  if you want it work well , change your width = height in yolo_*.cfg. I think yolo can not work good in this issue. so any one have good idea about this.

**Comment by dm0mb** [comment]
yolov4.cfg - 245 MB: yolov4.weights (Google-drive mirror yolov4.weights ) paper Yolo v4 just change width= and height= parameters in yolov4.cfg file and use the same yolov4.weights file for all cases:

width=608 height=608 in cfg: 65.7% mAP@0.5 (43.5% AP@0.5:0.95) - 34(R) FPS / 62(V) FPS - 128.5 BFlops
width=512 height=512 in cfg: 64.9% mAP@0.5 (43.0% AP@0.5:0.95) - 45(R) FPS / 83(V) FPS - 91.1 BFlops
width=416 height=416 in cfg: 62.8% mAP@0.5 (41.2% AP@0.5:0.95) - 55(R) FPS / 96(V) FPS - 60.1 BFlops
width=320 height=320 in cfg: 60% mAP@0.5 ( 38% AP@0.5:0.95) - 63(R) FPS / 123(V) FPS - 35.5 BFlops

**Comment by Prasanna-icefire** [comment]
I managed to solve this issue by changing the subdivisions=16 to subdivisions=64 in my yolo-obj.cfg file.

**Comment by Cococatty** [comment]
Hey all, I cloned the repo today and having the same issue. 

I tried the ImageNet, Cifar-10 (Classifier) examples on darknet official site but neither of them are working.   :( 

Error message as below:
First section must be [net] or [network]: Success
darknet: ./src/utils.c:256: error: Assertion `0' failed.
Aborted (core dumped)

I have re-cloned the repo but not helping.

Any advice?

Thanks! :D

**Comment by thunfischtoast** [comment]
Here is something to try, it solved the problem for me: https://github.com/pjreddie/darknet/issues/235#issuecomment-340679060

TLDR: When copying config files from windows to linux, run `dos2unix` on it before continuuing.

**Comment by sba17** [comment]
Updated subdivision to 64 has resolved the issue.

**Comment by hoffm621** [comment]
> In my case, that was the problem with images width and height, please check.

Can you tell me exactly how you fixed the height and width problems?
